script/functions.sh

```diff
@@ -300,40 +300,46 @@ function maven_build_jar() {
     return 0
 }
 
+function build_module() {
+    local module=$1
+    local setting_xml_path="${ODC_DIR}/builds/settings.xml"
```

Comment:
there is no file named this `${ODC_DIR}/builds/settings.xml`

Reply:
I have changed scripts that names are build_libs.sh and function.sh